packages/imagelib/imagelib.20170118/descr

```diff
@@ -0,0 +1,17 @@
+The imagelib library implements image formats such as PNG or PPM
+
+The imagelib library implements image formats such as PNG or PPM  in
+OCaml, relying on only one external dependency: camlzip. However, we
+plan to reimplement zlib in OCaml at some point.
```

Comment:
You may be interested in http://ocamllabs.io/rfc/2017/07/27/NewGitRFC.html by @dinosaure, which also includes a separate library that is a pure implementation of parts of zlib